packages/server/src/events/tasks.ts

```diff
@@ -32,6 +32,36 @@ export async function DistributeArticleStory(email: string, articleId: number) {
  */
 export async function AddSuggestedUserToFollow(email: string, articleId: number) {
     // email is the email id of logged in user
+    const prisma = PrismaClientSingleton.prisma;
+    const article = await prisma.article.findUnique({
+        where: {
+            id: articleId,
+        },
+        include: {
+            user: {
+                select: {
+                    id: true,
+                },
+            },
+        },
+    });
+
+    const authorId = article?.user.id!
+
+    const updatedUser = await prisma.user.update({
```

Comment:
Once a logged-in user takes a positive action such as liking, commenting, or saving an article, we aim to suggest the article's author as someone to follow. It's important to note that this is a suggestion, and there is no direct connection between the author and the user who performed the action, hence the property 'isSuggested: true.' Additionally, this suggested author will be displayed as a user to follow for the logged-in user. As a result, this author will be added directly to the following list of the logged-in user, similar to how Instagram's followers and following work when you click 'follow,' and it gets added to your following list. I appreciate the other changes you've made, but please update this part and resubmit the PR. Thank you.

Reply:
`User` to `Follower` is a one-to-many relation. How to know which `Follower` to update? or to update all `Followers`?

Reply:
Use `followings` property of user to connect the author.

Reply:
This is not correct. Please use this one and learn what is going on with this code. Thanks 

```js
export async function AddSuggestedUserToFollow(email: string, articleId: number) {
    // email is the email id of logged in user
    // since we need author id we need to fetch it ( only article is is provided )
    const prisma = PrismaClientSingleton.prisma;
    const article = await prisma.article.findUnique({
        where: {
            id: articleId,
        },
        include: {
            user: {
                select: {
                    id: true,
                },
            },
        },
    });

    if (!article) return;
    const authorID = article.user.id;

    await prisma.user.update({
        where: {
            email: email,
        },
        data: {
            followings: {
                create: {
                    followed: {
                        connect: {
                            id: authorID,
                        },
                    },
                    isSuggested: true,
                },
            },
        },
    });
}
```